.github/workflows/e2e.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.7", "3.12" ]
+        python-version: [ "3.8", "3.12" ]
```

Comment:
Why? We haven't changed our minimum Python version requirements for Python client.

Reply:
The figure class has a different name in py3.7. I can update the condition to handle both, but should we add debt to support a legacy python version?

Reply:
It doesn't sound to be a huge one so why not? 😉; at least such a small change shouldn't impact our matrix. You've could think to skip the tests for particular OS/Python combination.

Reply:
@PatrykGala - are we good to exclude py3.7?
In that case, I'll update the changelog to mention that we are officially dropping support for py3.7

Reply:
As discussed internally, dropping py3.7 support